tools/stress-cluster/cluster/kubernetes/stress-test-addons/images/test-resource-deployer/test-resources-post.ps1

```diff
@@ -12,22 +12,47 @@ if ([string]::IsNullOrEmpty($outputFile)) {
 }
 
 $duplicates = @{}
-$environmentText = ''
+$envLines = @()
 
 try {
     foreach ($line in (Get-Content $outputFile)) {
         $duplicates[$line] = $true
-        $environmentText += ($line + "`n")
+        $envLines += $line
     }
-} catch {}
+}
+catch {}
 
 foreach ($entry in $DeploymentOutputs.GetEnumerator()) {
     $line = "$($entry.name)=$($entry.value)"
     if (!$duplicates.Contains($line)) {
         $duplicates[$line] = $true
-        $environmentText += ("$line" + "`n")
+        $envLines += $line
     }
 }
 
+$environmentText = $envLines -join "`n"
```

Comment:
I assume no trailing newline is going to break anything here but just putting in a note to remember.

Reply:
Fixed.